lib/deploy/storage/prepare.js

```diff
@@ -1,29 +1,21 @@
 'use strict';
 
+var chalk = require('chalk');
 var fs = require('fs');
 var RSVP = require('rsvp');
-var api = require('../../api');
+
+var gcp = require('../../gcp');
 var utils = require('../../utils');
-var chalk = require('chalk');
 
-module.exports = function(context, options, payload) {
+module.exports = function(context, options) {
   var rulesPath = options.config.get('storage.rules');
   if (rulesPath) {
```

Comment:
If `rulesPath` is not defined, is there a message to the user?

Reply:
No, and I don't think we want one (see above comment).